pkg/agent/route/route_linux.go

```diff
@@ -665,6 +751,17 @@ func (c *Client) restoreIptablesData(podCIDR *net.IPNet,
 
 	writeLine(iptablesData, "*filter")
 	writeLine(iptablesData, iptables.MakeChainLine(antreaForwardChain))
+
+	for _, chain := range c.fixedNodeNetworkPolicyChains {
+		writeLine(iptablesData, iptables.MakeChainLine(chain))
+	}
+	for chain := range nodeNetWorkPolicyIPTables {
+		// Skip these two chains since they are included in fixedNodeNetworkPolicyChains.
```

Comment:
how about make `c.fixedNodeNetworkPolicyChains` a set and only create a chain when the set doesn't include it? want to avoid business specific code here as much as possible.

or it could even merge fixedNodeNetworkPolicyIPTablesIPv4, fixedNodeNetworkPolicyIPTablesIPv6, and fixedNodeNetworkPolicyChains to nodeNetworkPolicyIPTablesIPv4 and nodeNetworkPolicyIPTablesIPv6, so it doesn't need to handle two cases and doesn't have to deduplicate here.